apps/web/core/hooks/organizations/teams/use-public-organization-teams.ts

```
@@ -23,96 +19,137 @@
-				let responseTasks = (res.data.tasks as ITask[]) || [];
-				if (Array.isArray(responseTasks) && responseTasks.length > 0) {
-					responseTasks = responseTasks.map((task) => {
-						const clone = cloneDeep(task);
-						if (task.tags && task.tags?.length) {
-							clone.label = task.tags[0].name;
-						}
+			let responseTasks = publicTeamData.tasks || [];
+			if (Array.isArray(responseTasks) && responseTasks.length > 0) {
+				responseTasks = responseTasks.map((task) => {
+					const clone = cloneDeep(task);
+					if (task.tags && task.tags?.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)